.github/dependabot.yml

```diff
@@ -21,7 +21,7 @@ updates:
 
   - package-ecosystem: "pip"
     target-branch: "main"
-    directory: "/client_cli/pytests/"
+    directory: "/iroha_cli/pytests/"
```

Comment:
can you remind what was the conclusion, why not merge this binary with it's library?

Reply:
We don't want to force users to pull in CLI dependencies, and if we separate them with optional features, building or installing the binary would be even less convenient because the feature flag must be specified.